### PR TITLE
Add Korean(ko) translation

### PR DESCRIPTION
**Pull request by potatogim**
Add Korean(ko) translation

Signed-off-by: Ji-Hyeon Gim <potatogim@gluesys.com>

### CODE REVIEW

File: i18n/ko.toml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  [socials.github]
+  other = "Github"
+
+[copyright]
+  [copyright.line1]
+  other = "© 2020 The Rocky Linux Project. All rights reserved."
+
+  [copyright.line2]
+  other = "CentOS는 Rd Hat, Inc.의 등록 상표입니다. Rocky Linux 프로젝트는 Red Hat, Inc.와 제휴하고 있지 않고, Red Hat, Inc.가 보증하지 않습니다."
```
Comment (Lastorder-DC) [inline i18n/ko.toml]:
```suggestion
  other = "CentOS는 Red Hat, Inc.의 등록 상표입니다. Rocky Linux 프로젝트는 Red Hat, Inc.와 제휴하고 있지 않고, Red Hat, Inc.가 보증하지 않습니다."
```

File: i18n/ko.toml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  other = "Rocky Linux"
+
+  [landing.p1.content]
+  other = "커뮤니티 중심의 기업을 위한 운영 환경을 지원하는 리눅스 배포판입니다."
+
+  [landing.p2.heading]
+  other = "Rocky Linux 프로젝트가 뭐죠?"
+
+  [landing.p2.content]
+  other = "다운스트림 파트너가 방향 전환을 한 지금에 있어서, Rocky Linux는 미국 최고의 엔터프라이즈 리눅스 배포판과 100% 호환성을 갖도록 설계된 커뮤니티 엔터프라이즈 운영 체제입니다. Rocky Linux는 커뮤니티에서 집중적으로 개발하고 있으며, CentOS 프로젝트의 창립자인 Gregory Kurtzer가 이를 이끌고 있습니다. 릴리즈에는 ETA가 없으며, 기여자는 이 사이트에서 제공하는 커뮤니케이션 방법을 사용하여 연락해야 합니다."
```
Comment (Lastorder-DC) [inline i18n/ko.toml]:
```suggestion
  other = "다운스트림 파트너가 방향 전환을 한 지금에 있어서, Rocky Linux는 미국 최고의 엔터프라이즈 리눅스 배포판과 100% 호환성을 갖도록 설계된 커뮤니티 엔터프라이즈 운영 체제입니다. Rocky Linux는 커뮤니티에서 집중적으로 개발하고 있으며, CentOS 프로젝트의 창립자인 Gregory Kurtzer가 이를 이끌고 있습니다. 출시일은 아직 미정이며, 기여자는 이 사이트에서 제공하는 커뮤니케이션 방법을 사용하여 연락해야 합니다."
```

**Review by Lastorder-DC — COMMENTED** [review]
Looks great but there are some thing to fix

**Comment by Lastorder-DC** [comment]
#33

**Comment by FoggyMtnDrifter** [comment]
You also need to make changes to `config.toml` and `vercel.json` with your language code. Please make those changes and we can proceed with this one.

**Comment by potatogim** [comment]
> You also need to make changes to `config.toml` and `vercel.json` with your language code. Please make those changes and we can proceed with this one.

Thank you :smiley:

**Comment by FoggyMtnDrifter** [comment]
Can you fix the conflicts that exist and we can proceed.

**Comment by potatogim** [comment]
> Can you fix the conflicts that exist and we can proceed.

Sure. resolved!

**Pull request merged**